pkg/generate/app/imageref.go

```diff
@@ -288,18 +288,17 @@ func (r *ImageRef) DeployableContainer() (container *kapi.Container, triggers []
 		imageChangeParams := &deployapi.DeploymentTriggerImageChangeParams{
 			Automatic:      true,
 			ContainerNames: []string{name},
-			Tag:            tag,
 		}
 		if r.Stream != nil {
 			imageChangeParams.From = kapi.ObjectReference{
-				Kind:      "ImageStream",
-				Name:      r.Stream.Name,
+				Kind:      "ImageStreamTag",
```

Comment:
Please generate output from `new-app` for both of these old cases and make a test that demonstrates that they still work through conversion to produce the expected output.